packages/table/src/Table.story.tsx

```diff
@@ -739,6 +739,134 @@ export const SelectableRows: StoryFn<StoryTableProps> = args => {
   );
 };
 
+export const SelectableRowsWithNoHeader: StoryFn<StoryTableProps> = args => {
```

Comment:
this name is a bit confusing, since there's still a header row. I'd change it to `SelectableNoSelectAll` or similar